src/extensions/anvil/prop.ts

```diff
@@ -36,26 +34,24 @@ interface MembraneOrientation {
     readonly centroid: Vec3
 }
 
-namespace MembraneOrientation {
-    export enum Tag {
-        Representation = 'membrane-orientation-3d'
-    }
-
-    const pos = Vec3();
-    export const symbols = {
+export const MembraneOrientation = {
+    Tag: {
+        Representation: 'membrane-orientation-3d'
+    },
+    symbols: {
         isTransmembrane: QuerySymbolRuntime.Dynamic(CustomPropSymbol('computed', 'membrane-orientation.is-transmembrane', Type.Bool),
             ctx => {
                 const { unit, structure } = ctx.element;
                 const { x, y, z } = StructureProperties.atom;
                 if (!Unit.isAtomic(unit)) return 0;
                 const membraneOrientation = MembraneOrientationProvider.get(structure).value;
                 if (!membraneOrientation) return 0;
-                Vec3.set(pos, x(ctx.element), y(ctx.element), z(ctx.element));
+                Vec3.set(Vec3(), x(ctx.element), y(ctx.element), z(ctx.element));
```

Comment:
`pos` should be reused, e.g., create it at the module/file level

Reply:
After re-reading the code, it seems that `pos` is only used inside of `MembraneOrientation`'s `ctx` function? If so, then declaring the `pos` inside of that function should work?

```
ctx => {
    ...           

    const pos = Vec3();
    Vec3.set(pos, x(ctx.element), y(ctx.element), z(ctx.element));

    ...

    return isInMembranePlane(pos, normalVector, planePoint1, planePoint2);
})
```